v3/src/components/graph/components/scatterdots.tsx

```diff
@@ -292,9 +285,20 @@ export const ScatterDots = observer(function ScatterDots(props: PlotProps) {
           })
       }
     })
+    // Decrease point size when Connecting Lines are first activated so the lines are easier to see, and
+    // revert to original point size when Connecting Lines are deactivated.
+    if (!connectingLinesActivatedRef.current && showConnectingLines && pointSizeMultiplier > .5) {
+      origPointSizeMultiplier.current = pointSizeMultiplier
+      await pixiPointsRef.current?.setAllPointsScale(.5, transitionDuration)
+      graphModel.pointDescription.setPointSizeMultiplier(pointSizeMultiplier * .5)
+    } else if (!showConnectingLines) {
+      const scaleFactor = origPointSizeMultiplier.current / pointSizeMultiplier
+      await pixiPointsRef.current?.setAllPointsScale(scaleFactor, transitionDuration)
+      graphModel.pointDescription.setPointSizeMultiplier(origPointSizeMultiplier.current)
+    }
   }, [dataConfiguration, dataTip, dataset?.collections, graphModel.pointDescription,
-    handleConnectingLinesClick, handleConnectingLinesMouseOut, handleConnectingLinesMouseOver, layout,
-    showConnectingLines])
+      handleConnectingLinesClick, handleConnectingLinesMouseOut, handleConnectingLinesMouseOver, layout,
+      showConnectingLines, pointSizeMultiplier, connectingLinesActivatedRef, pixiPointsRef])
```

Comment:
```suggestion
      handleConnectingLinesClick, handleConnectingLinesMouseOut, handleConnectingLinesMouseOver,
      layout, pixiPointsRef, pointSizeMultiplier, showConnectingLines])
```
Don't need `connectingLinesActivatedRef`; others may as well be alphabetized.